apps/examples/testcase/ta_tc/systemio/itc/itc_uart.c

```diff
@@ -774,6 +774,76 @@ static void itc_systemio_iotbus_uart_set_flow_mode_flush_write_read_p(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
```

Comment:
I think it is better to separate this commit to implementations of iotbus functions and TCs.